src/resources/breaking-changes.md

```diff
@@ -0,0 +1,818 @@
+---
+title: Breaking changes
+description: A list of breaking changes by release in Dart.
+---
+
+This page lists the breaking changes in Dart, organized by release and area.
+Complete release notes are available in the [Dart SDK changelog][changelog].
+
+According to Dart's [breaking change policy][], each listed change contains a link to more
+information on the change, such as migration guidance, background, reasoning,
+caveats, and more.
+
+{{site.alert.note}}
+  Changes prior to Dart 3.1.0 and Dart's updated breaking change policy 
+  might not have links to additional information. 
+  More details on some of those changes might
+  be available in their corresponding [changelog][] entry.
+{{site.alert.end}}
+
+**Dart has two types of breaking changes**:
+* **Standard**: Causes breaks as soon as you [upgrade your sdk version][sdk]. 
+  *These are the majority of changes and are not specially marked in this list.*
+* **Language versioned**: Causes breaks only when you upgrade the
+  [language version] of your code. *These are marked "**Language versioned**"*.
+
```

Comment:
If we cover deprecations here, we should probably have a comment and marker about deprecations.

Reply:
I added this bullet for deprecations, but am not sure it's correct:

```md
* **Deprecations**: The compiler maintains compatibility for deprecated code,
  with a warning. Deprecations are then completely removed in the release
  following their deprecation, breaking any code that relies on the previous behavior.
  
  *These are marked "**Deprecated**" and "**Removed**", respectively*.
```

Reply:
I was also wondering, going back through older entries, is _anything_ that even mentions "removed/removal/etc" considered a removal, based on my definition above?

Reply:
> is anything that even mentions "removed/removal/etc" considered a removal

I think this is case to case.  Someone might mention removing something in the CHANGELOG that still wasn't part of the public API.